models/team_todolist.js

```diff
@@ -0,0 +1,43 @@
+module.exports = (sequelize, DataTypes) => {
+    const team_todolist = sequelize.define("team_todolist", {
+        todolist_ID: {
+            type: DataTypes.STRING,
+            primaryKey: true,
+            allowNull: false
+        },
+        team_ID:{
+            type: DataTypes.STRING,
+            allowNull: true
+        },
+        name: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        description: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        state:{
+            type: DataTypes.INTEGER,
+            allowNull: false
+        },
```

Comment:
What is state?
Why type is integer?

Reply:
State indicate which color todolist  should display right now according to discussion with front-end, there is 3 color (green, yellow, red) so I make it integer instead of boolean or should I remove it and let front-end manage it with datetime that keep in?

Reply:
I see.
You can keep it as integer or [enum](https://sequelize.org/master/class/lib/data-types.js~ENUM.html).